File: frontend/todolist/src/app/model/list-item.ts
export class ListItem {
  id: number;
  title: string;
  description: string;
  doBy: string;

  constructor(id: number, title: string, description: string, doBy: string) {
    this.id = id;
    this.title = title;
    this.description = description;
    this.doBy = doBy;
  }
}

File: backend/todolist/src/main/java/todolist/models/ListItem.java
package todolist.models;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import javax.persistence.*;
import java.time.LocalDate;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Table(name = "listitems")
public class ListItem {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String title;
    private String description;
    @Column(name = "doby")
    private String doBy;

    @Override
    public String toString() {
        return "ListItem{" +
                "id=" + id +
                ", title='" + title + '\'' +
                ", description='" + description + '\'' +
                ", doby=" + doBy +
                '}';
    }
}
